src/patterns/confirmation-pages/default.md.njk

```diff
@@ -4,18 +4,24 @@ layout: layout-example.njk
 
 {% from "panel/macro.njk" import govukPanel %}
 
-{{ govukPanel({
-  "titleText": "Application complete",
-  "html": "Your reference number<br><strong>HDJ2123F</strong>"
-})
-}}
+<div class="govuk-o-main-wrapper">
+  <div class="govuk-o-grid">
+    <div class="govuk-o-grid__item govuk-o-grid__item--two-thirds">
+  {{ govukPanel({
+    "titleText": "Application complete",
+    "html": "Your reference number<br><strong>HDJ2123F</strong>"
+  })
+  }}
 
-<p class="govuk-body">We have sent you a confirmation email.</p>
+  <p class="govuk-body">We have sent you a confirmation email.</p>
```

Comment:
Can we sort out the indentation in this example?

Reply:
If I indent this properly all of the text turns into a code block

Reply:
can you do what I did with colours, and just make this a .njk file, not a .md.njk? then it won't get the code block issue

Reply:
Yeah, there's probably no reason for any of the examples to use the `.md` extension as none of them contain markdown…

Reply:
Fixed